Extract.hs

```diff
@@ -0,0 +1,28 @@
+import System.IO
+import System.Environment
+import Data.List
+
+main = do
+    args <- getArgs
+    header <- return( args !! 0)
+    file <- return( args !! 1)
```

Comment:
```suggestion
header:file:args <- getArgs
```
note that this will fail if enough args aren't passed. You could use `optparse-applicative` if the command line args get more complex, as an aside.